src/binary-reader.h

```diff
@@ -51,6 +51,12 @@ struct ReadBinaryOptions {
   bool fail_on_custom_section_error = true;
 };
 
+// TODO: Move somewhere else?
```

Comment:
Probably yes, though I couldn't say where. Seems reasonable to leave it here at least.